data/README.md

```diff
@@ -117,3 +122,17 @@ https://github.com/apache/parquet-format/commit/54839ad5e04314c944fed8aa4bc6cf15
 
 `bloom_filter.xxhash.bin` uses the newer xxHash-based bloom filter format as of
 https://github.com/apache/parquet-format/commit/3fb10e00c2204bf1c6cc91e094c59e84cefcee33.
+
+## NaN in stats
+
+Previous versions of the C++ Parquet writer would write NaN values in min and max
```

Comment:
"Previous versions" is quite unspecific, can you be more explciit?

Reply:
I noted the Parquet version that changed in.